test_data.sql

```diff
@@ -1,9 +1,15 @@
 DELETE FROM CourseEnrollments;
 DELETE FROM Courses;
 DELETE FROM Students;
+DELETE FROM Tutors;
+DELETE FROM TutorsAssignment;
 
-INSERT INTO Students(id, first_name, last_name) VALUES (3, "Giulia", "Contarino"), (2, "Tommaso", "Tranchese"), (1, "Federico", "Urbani");
+INSERT INTO Students(id, first_name, last_name) VALUES(7,"Lorenzo", "Catania"), (6,"Eduard","Ancuta"), (5,"Giada", "Dell'Ara"), (4,"Lucrezia","Rumore"), (3, "Giulia", "Contarino"), (2, "Tommaso", "Tranchese"), (1, "Federico", "Urbani");
```

Comment:
idem qui, i dati che aggiungi mettili in un file sotto la tua cartella personale